scripts/overview.json

```diff
@@ -4166,6 +4210,7 @@
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
           "decimals": 5,
+          "description": "the time consumed by TiKV queries",
```

Comment:
TiKV queries -> TiDB queries?

Reply:
The handle time of each Coprocessor request